secureli/actions/action.py

```diff
@@ -264,7 +264,9 @@ def _run_post_install_scan(
             self.action_deps.echo.print(f"running {secret_test_id}.")
 
             scan_result = self.action_deps.scanner.scan_repo(
-                folder_path, ScanMode.ALL_FILES, specific_test=secret_test_id
+                folder_path,
+                ScanMode.ALL_FILES,
+                specific_test=secret_test_id,
```

Comment:
I think the only change here was adding a comma? Did you mean to pass through the files variable here as well?

Reply:
Won't need it here since install should only run on all files